compute/cker/include/cker/operation/RmsNorm.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __NNFW_CKER_RMS_NORM_H__
+#define __NNFW_CKER_RMS_NORM_H__
+
+#include "cker/Shape.h"
+#include "cker/Types.h"
+#include "cker/Utils.h"
+
+#include <cmath>
+
+namespace nnfw
+{
+namespace cker
+{
+
+inline void RmsNorm(const RmsNormParams &params, const Shape &input_shape, const float *input_data,
+                    const Shape &gamma_shape, const float *gamma_data, const Shape &beta_shape,
+                    const float *beta_data, const Shape &output_shape, float *output_data)
+{
+  const int32_t batches = MatchingDim(input_shape, 0, output_shape, 0);
+  const int32_t heights = MatchingDim(input_shape, 1, output_shape, 1);
+  const int32_t widths = MatchingDim(input_shape, 2, output_shape, 2);
+  const int32_t channels = MatchingDim(input_shape, 3, output_shape, 3);
+
+  UNUSED_RELEASE(gamma_shape);
+  UNUSED_RELEASE(beta_shape);
```

Comment:
If it won't be used again, could you please remove it from the parameter list?

Reply:
> If it won't be used again, could you please remove it from the parameter list?

Okay, I'll remove it. :)

Reply:
> > If it won't be used again, could you please remove it from the parameter list?
> 
> Okay, I'll remove it. :)

I thought it's better to check gamma shape and beta shape in this code.
I'll fix it. :)